src/Data/Inputs.elm

```diff
@@ -117,12 +109,21 @@ setCustomCountryMix index value ({ customCountryMixes } as query) =
     }
 
 
+updatedCountryList : Result String Material -> List Country -> List Country.Code -> Result String (List Country)
+updatedCountryList material countriesDB countries =
+    material
+        |> Result.andThen
+            (\{ defaultCountry } ->
+                countries
+                    -- Update the list of countries: the first country (from the material step) is constrained to be the material's default country
+                    |> updateCountryList 0 defaultCountry
```

Comment:
How about calling `LE.setAt 0 defaultCountry` directly here? I don't see much added value of the `updateCountryList` alias here…

Reply:
It's also called [here](https://github.com/MTES-MCT/wikicarbone/pull/54/files#diff-43b5051efc2b0cb128ab779135733e99a5cc1ddd6a100f03f3896be52c9c7896R132), would you rather I change it to be a direct all in both places?

Reply:
Yes totally 👍